server.js

```diff
@@ -1,9 +1,11 @@
 // BASE SETUP
 var express = require('express');
 var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
The default port for mongo is 27017, and the only other port I've seen was technically an invalid port, but I'll pull the port into a separate variable within this same file, that we can move later.